includes/querypages/PropertiesQueryPage.php

```diff
@@ -276,7 +276,7 @@ function getResults( $requestOptions ) {
 	 * @return array|bool
 	 */
 	public function getQueryInfo() {
-		return false;
+		return null;
```

Comment:
This does not match with the above type doc

Reply:
Fixed, thanks, done.